source/common/stats/histogram_impl.h

```diff
@@ -97,17 +100,19 @@ class HistogramImpl : public HistogramImplHelper {
  */
 class NullHistogramImpl : public HistogramImplHelper {
 public:
-  explicit NullHistogramImpl(SymbolTable& symbol_table)
-      : HistogramImplHelper(symbol_table), symbol_table_(symbol_table) {}
+  NullHistogramImpl(SymbolTable& symbol_table, Unit unit)
+      : HistogramImplHelper(symbol_table), symbol_table_(symbol_table), unit_(unit) {}
   ~NullHistogramImpl() override { MetricImpl::clear(symbol_table_); }
 
   bool used() const override { return false; }
   SymbolTable& symbolTable() override { return symbol_table_; }
 
+  Unit unit() const override { return unit_; };
```

Comment:
i'm thinking just hard-code returning Unspecified here?

Reply:
See the reply on `thread_local_store.cc`.

Reply:
Fixed, pushing in a bit.